app/src/main/AndroidManifest.xml

```diff
@@ -9,6 +9,7 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.AppCompat.Light.NoActionBar">
+        <activity android:name=".activity.ReservationsRecyclerActivity"></activity>
```

Comment:
if there is nothing inside activity please use  "/>" instead of "></activity>"

Reply:
done